app/services/editions/db/FrontsQueries.scala

```diff
@@ -59,4 +73,86 @@ trait FrontsQueries extends Logging {
       isHidden
     }
   }
+
+  protected def getFront(frontId: String)(implicit session: DBSession): Option[EditionsFront] = {
+    val rows: List[FrontAndNestedEntitiesRow] =
+      sql"""
+        SELECT
+          fronts.id            AS fronts_id,
+          fronts.issue_id      AS fronts_issue_id,
+          fronts.index         AS fronts_index,
+          fronts.name          AS fronts_name,
+          fronts.is_special    AS fronts_is_special,
+          fronts.is_hidden     AS fronts_is_hidden,
+          fronts.metadata      AS fronts_metadata,
+          fronts.updated_on    AS fronts_updated_on,
+          fronts.updated_by    AS fronts_updated_by,
+          fronts.updated_email AS fronts_updated_email,
+
+          collections.id            AS collections_id,
+          collections.front_id      AS collections_front_id,
+          collections.index         AS collections_index,
+          collections.name          AS collections_name,
+          collections.is_hidden     AS collections_is_hidden,
+          collections.metadata      AS collections_metadata,
+          collections.updated_on    AS collections_updated_on,
+          collections.updated_by    AS collections_updated_by,
+          collections.updated_email AS collections_updated_email,
+          collections.prefill       AS collections_prefill,
+          collections.path_type     AS collections_path_type,
+          collections.content_prefill_window_start       AS collections_content_prefill_window_start,
+          collections.content_prefill_window_end         AS collections_content_prefill_window_end,
+
+          cards.collection_id AS cards_collection_id,
+          cards.id            AS cards_id,
+          cards.card_type     AS cards_card_type,
+          cards.index         AS cards_index,
+          cards.added_on      AS cards_added_on,
+          cards.metadata      AS cards_metadata
```

Comment:
that's a lorra lorra important strings - I guess it's not possible to select on a `.` at line 119 and that's why we have to rename them all?

Reply:
I know eh 😢  – each joined table is scoped to its entity to prevent collisions. We could share this string with `IssueQueries`, though! Not sure I understand the `.` suggestion, but keen to reduce repetition if poss.